cfgov/templates/wagtailadmin/login.html

```diff
@@ -1,10 +1,9 @@
 {% overextends "wagtailadmin/login.html" %}
-{% load compress static i18n %}
-
+{% load static i18n %}
```

Comment:
Same comment re `static` instead of [`staticfiles`](https://github.com/wagtail/wagtail/blob/v1.13.1/wagtail/wagtailadmin/templates/wagtailadmin/login.html#L2).